modules/catalog/src/main/java/org/apache/ignite/internal/catalog/events/AlterColumnEventParameters.java

```diff
@@ -23,32 +23,36 @@
  * Create table event parameters contains a column descriptor for the modified column.
  */
 public class AlterColumnEventParameters extends CatalogEventParameters {
-
     private final int tableId;
 
-    private final CatalogTableColumnDescriptor columnDescriptor;
+    private final CatalogTableColumnDescriptor descriptor;
```

Comment:
And here. These are changes for the sake of changes

Reply:
But not for me.